docs/configuration.md

```diff
@@ -1033,3 +1033,67 @@ fail-fast matrices and support is more limited because of that expectation.
     ```
     failFast.set(true)
     ```
+
+
+### additionalFlankConfig
+Allow appending additional config to flank root yaml. This option is useful when you would like to test option
+before it is available on Fladle. Supports both single and multiple properties.
+
+Single property
+
+=== "Groovy"
+```
+additionalFlankConfig = "new-property: true"
+```
+
+=== "Kotlin"
+```
+additionalFlankConfig.set("new-property: true")
+```
+
+Multiple properties
```

Comment:
Thanks for adding the single and multiple property examples. Can you make sure this looks right? You can test the documentation with `mkdocs serve`.
```suggestion
Multiple options
```

Reply:
works good